### PR TITLE
Adds release note about rsync distributors.

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

File: docs/user-guide/release-notes/2.10.x.rst [hunk 1]
```diff
@@ -15,3 +15,6 @@ New Features
 * A full re-sync can be forced with a 'force_full' option. It will trigger full sync where
   no sync optimization will be applied or steps skipped. This option can be used by ``force_full``
   option in override_config via API or ``--force-full`` via CLI.
+
+* RPM, ISO, and Docker repositories can be published using rsync distributors. These distributors
```
Comment (bmbouter) [inline docs/user-guide/release-notes/2.10.x.rst]: s/using rsync/using the new rsync/

**Comment by bmbouter** [comment]
The one comment is a suggestion, but I'm going to add the LGTM now so you can handle it how you would like. @dkliban thanks for the release note.

**Comment by bmbouter** [comment]
The one comment is a suggestion, but I'm going to add the LGTM now so you can handle it how you would like. @dkliban thanks for the release note.

**Comment by dkliban** [comment]
I fixed the wording.

**Pull request merged**